Kevin/diagrams

kl-kevin opened the pull request.

shakeelmohamed commented:
Looks good! Can we also update the readme here: https://github.com/seattleu/exbedia-admin-app-frontend

kl-kevin commented:
The images was removed when the branch was deleted, so I updated the urls to linked to the images stored on master branch. Also I have updated the readme on https://github.com/seattleu/exbedia-admin-app-frontend
